src/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -785,4 +785,12 @@ The header is followed by the list of parameters and their errors (might be seve
     <value>To install the template package(s) anyway, apply '{0}' option:</value>
     <comment>{0} is option to use (--force). Followed by command to use to install the packages.</comment>
   </data>
+  <data name="TemplatePackageCoordinator_BuiltInCheck_Info_BuiltInPackageAvailable" xml:space="preserve">
+    <value>An update for template package '{0}' is available from built-in provider.</value>
```

Comment:
For this message - do we have any idea _which_ built-in provider has the template match? It would be great to be able to say something like:

> An update for template package Boxed.Templates::1.0.0 is available in the .NET SDK template packages.
> To use this built-in template package, uninstall the manually installed template package using:
>     dotnet new uninstall ........

where the phrase `'the .NET SDK template packages'` is keyed off of the provider id or something. That sounds natural to me, and it also gives us some traceability/debugging aids if users report this message and want clarification.

Reply:
Yes, that's possible. 
We just need to put meaningful names here:

https://github.com/dotnet/sdk/blob/efd4338e20b302e9ac11cebc7bc2a5a5bb5dd349/src/Cli/dotnet/commands/dotnet-new/OptionalWorkloadProviderFactory.cs#L18

https://github.com/dotnet/sdk/blob/efd4338e20b302e9ac11cebc7bc2a5a5bb5dd349/src/Cli/dotnet/commands/dotnet-new/BuiltInTemplatePackageProviderFactory.cs#L18

Just wondering if we need a word 'provider' afterwards: `.NET SDK template packages provider`, where `.NET SDK template packages` is coming from factory. 
Same with optional workloads `Optional Workloads provider`.  

Or `is available in '.NET SDK template packages' provider`.

Reply:
Names are adapted here: https://github.com/dotnet/templating/pull/4514
I went for `.NET SDK` as it is shorter and clear enough, we will likely start using them in tabular output at some point so size matters.

Option A:
```
An update for template package Boxed.Templates::1.0.0 is available in .NET SDK.
An update for template package Boxed.Templates::1.0.0 is available in Optional workloads.
```

Option B: ("template packages" is part of the string)
```
An update for template package Boxed.Templates::1.0.0 is available in .NET SDK template packages.
An update for template package Boxed.Templates::1.0.0 is available in Optional workloads template packages.
```

Option C (safest in terms of further changes):
```
An update for template package Boxed.Templates::1.0.0 is available in '.NET SDK' provider.
An update for template package Boxed.Templates::1.0.0 is available in 'Optional workloads' provider.
```

Reply:
I think that makes sense,  'provider' is a good addition.

Reply:
I like C with a small tweak - 'the <provider name> provider' is the most correct grammar, and it lets us keep the shorter names for tabular output.